plugin/trino-hive/src/test/java/io/trino/plugin/hive/benchmark/BenchmarkFileFormat.java

```diff
@@ -0,0 +1,42 @@
+/*
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package io.trino.plugin.hive.benchmark;
+
+/**
+ * File formats from StandardFileFormats class captured in an enum for JMH benchmarks
+ */
+public enum BenchmarkFileFormat
```

Comment:
why we need this enum since we have `StandardFileFormats`?

`BenchmarkFileFormat` is not extendable